src/command.rs

```diff
@@ -280,3 +280,36 @@ impl Command {
     protocol.set(o, self.addr, &bytes).map_err(Into::into)
   }
 }
+
+#[cfg(test)]
+mod tests {
+  use super::*;
+
+  #[test]
+  fn parse_double_from_int() {
+    let command = Command {
+      addr: 0x0886,
+      mode: AccessMode::Read,
+      data_type: DataType::Double,
+      parameter: Parameter::Int,
+      block_count: None,
+      block_len: 4,
+      byte_len: 4,
+      byte_pos: 0,
+      bit_len: None,
+      bit_pos: 0,
+      conversion: Some(Conversion::SecToHour),
+      lower_bound: None,
+      upper_bound: None,
+      unit: Some("h"),
+      mapping: None,
+    };
+    let buf = vec![0; command.block_len];
+    let value = command.parse_value(&buf, &[0x00, 0x95, 0xBA, 0x0A]).unwrap();
+    if let Value::Double(n) = value {
+      assert_eq!(n, 50000.0);
+    } else {
+      unreachable!("wrong datatype");
```

Comment:
Might be good to implement/derive `PartialEq` for `Value`.

Reply:
I tried that first, I had to add `PartialEq` to all data types and failed nevertheless at some point.

Reply:
I see you added `PartialEq` [here](https://github.com/reitermarkus/vcontrol-rs/pull/13/commits/28160210db89c86a4b9563e8ee33189029fc9d9f). I will update this PR with `assert_eq!(value, Value::Double(50000.0));` once you merged the async branch.